RELEASES.md

```diff
@@ -1,3 +1,135 @@
+- [`<float>::min`](https://doc.rust-lang.org/stable/std/primitive.f32.html#method.min)
+- [`<float>::clamp`](https://doc.rust-lang.org/stable/std/primitive.f32.html#method.clamp)
+- [`<float>::abs`](https://doc.rust-lang.org/stable/std/primitive.f32.html#method.abs)
+- [`<float>::signum`](https://doc.rust-lang.org/stable/std/primitive.f32.html#method.signum)
+- [`<float>::copysign`](https://doc.rust-lang.org/stable/std/primitive.f32.html#method.copysign)
+- [`MaybeUninit::write`](https://doc.rust-lang.org/stable/std/mem/union.MaybeUninit.html#method.write)
+
+
+<a id="1.85.0-Cargo"></a>
+
+Cargo
+-----
+- [Add future-incompat warning against keywords in cfgs and add raw-idents](https://github.com/rust-lang/cargo/pull/14671/)
+- [fix(cargo-rustc): stabilize higher precedence trailing flags](https://github.com/rust-lang/cargo/pull/14900/)
+- [feat(build-script): Pass CARGO_CFG_FEATURE ](https://github.com/rust-lang/cargo/pull/14902/)
+
+<a id="1.85.0-Rustdoc"></a>
+
+Rustdoc
+-----
+- [Doc comment on impl blocks shows the first line, even when the impl block is collapsed](https://github.com/rust-lang/rust/pull/132155)
+
+
+<a id="1.85.0-Compatibility-Notes"></a>
+
+Compatibility Notes
+-------------------
+- [`rustc` no longer treats the `test` cfg as a well known check-cfg](https://github.com/rust-lang/rust/pull/131729), instead it is up to the build systems and users of `--check-cfg`[^check-cfg] to set it as a well known cfg using `--check-cfg=cfg(test)`.
+  This is done to enable build systems like Cargo to set it conditionally, as not all source files are suitable for unit tests.
+  [Cargo (for now) unconditionally sets the `test` cfg as a well known cfg](https://github.com/rust-lang/cargo/pull/14963).
+[^check-cfg]: https://doc.rust-lang.org/nightly/rustc/check-cfg.html
+- [Disable potentially incorrect type inference if there are trivial and non-trivial where-clauses](https://github.com/rust-lang/rust/pull/132325)
+- `std::env::home_dir()` has been deprecated for years, because it can give surprising results in some Windows configurations if the `HOME` environment variable is set (which is not the normal configuration on Windows). We had previously changed its behavior, out of concern for compatibility with code depending on this non-standard configuration. Given how long this function has been deprecated, we're now fixing its behavior as a bugfix, and the next version will undeprecate it. (We deferred undeprecating it on nightly until today's stable release, to avoid potential confusion between the fixed version on nightly and the previously unfixed version on stable.)
+- [More closely match clang behavior for definition of core::ffi::c_char](https://github.com/rust-lang/rust/pull/132975)
+  On many targets, this has changed the definition of c_char, resulting in potential compilation issues.
+  The new definition is more likely to be accurate for the corresponding C definition on the relevant target.
```

Comment:
Question: does this actually mean the "platform default C definition"?

Reply:
I don't know what the actual change did, this is a best effort description. I don't mind clarifying, but it seems good to ask on the relevant tracking issue (https://github.com/rust-lang/rust/issues/134157) to centralize discussion and reduce noise if we ping folks to find out.

Reply:
I think it may be important to mention that this this doesn't affect much than T2/T3 no_std so most people can dismiss it quickly. Updated the notes in https://github.com/rust-lang/rust/issues/134157 to mention that, and slightly clarify the above.